### ISSUE TITLE
AP is not displayed

### ISSUE THREAD

**Issue by JefMore** [scott-vincent/instrument-panel#38]
Hello, great job you did, my question is why the autopilot is not showing in NAV, as it appears in bitmaps

**Comment by scott-vincent** [comment]
Thanks. Do you mean the autopilot that you can see in nav.png? There are two reasons why this wouldn't display. First, the Nav instrument is disabled by default as I now use the separate panel you can see here: https://github.com/scott-vincent/autopilot-panel If you want to enable the Nav instrument you need to edit your settings file (settings/instrument-panel.json) and swap out one of the other instruments for the Nav one (set Nav enabled to true and set the position and size). Secondly, the Nav instrument has 2 pages and it shows the Radio one by default, not the Autopilot one. You would need a rotary encoder to be able to changes pages, or you could edit the code and swap the following 2 lines in instrument-panel/instruments/nav.cpp:

if ((int)switchSel < (int)Autopilot) {
    renderNav();
}
else {
    renderAutopilot();
}

**Comment by JefMore** [comment]
Thanks Scott for your prompt reply, I'm sorry to tell you that my programming skills are very low. In instrument-panel.json, NAV is set to true. I already checked the part of the code that you indicate to change. I ask: is this how it should look?
if ((int)Autopilot < (int)switchSel {
renderAutopilot();
}
else {
renderNav();
}

**Comment by scott-vincent** [comment]
No, you swapped the 2 lines but you also swapped the if statement around. Just remove all those lines and replace with:
renderAutopilot();

**Comment by JefMore** [comment]
Thank you very much Scott for sharing your knowledge. now the autopilot is displayed
